### PR TITLE
fix on comments

### PR DESCRIPTION
**Pull request by k-okada**

### CODE REVIEW

**Comment by scpeters** [comment]
Did you mean to target `ros/common_tutorials`?

**Comment by k-okada** [comment]
No
If you merge this, then your following PR will be updated
https://github.com/ros/common_tutorials/pull/14

2015年12月1日火曜日、Steven Petersnotifications@github.comさんは書きました:

> Did you mean to target ros/common_tutorials?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/scpeters/common_tutorials/pull/2#issuecomment-160792327
> .

## 

## 

◉ Kei Okada

**Comment by scpeters** [comment]
sorry, I forgot about that pull request!

File: actionlib_tutorials/scripts/fibonacci_client.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #! /usr/bin/env python
 
 import rospy
+from __future__ import print_function
```
Comment (dirk-thomas) [inline actionlib_tutorials/scripts/fibonacci_client.py:4]: This line must occur before any other line, see #16.

**Pull request merged**